scripts/babel/transform-object-assign-require.js

```diff
@@ -7,12 +7,17 @@
 
 'use strict';
 
-module.exports = function autoImporter(babel) {
-  function getAssignIdent(path, file, state) {
+const declare = require('@babel/helper-plugin-utils').declare;
+const addDefault = require('@babel/helper-module-imports').addDefault;
+
+module.exports = declare(api => {
+  api.assertVersion(7);
+
+  function getAssignIdent(path, state) {
     if (state.id) {
       return state.id;
```

Comment:
For what it's worth, I had to update my local copy of this transform as follows: https://gist.github.com/NMinhNguyen/4c3f8fdbd99730df6e00e5c33a23751c. This is against beta 42 so things might have changed since then, but I definitely encountered problems without doing `babel.types.cloneNode(state.id)`